css/responsive/desktop.styl

```diff
@@ -1,5 +1,15 @@
 // ******************************************************
 
+// FORMS
+
+input
```

Comment:
because this is going to be the basis for any future form elements as they will share styling